src/shared-components/selectorButton/index.js

```diff
@@ -26,10 +27,12 @@ const propTypes = {
 };
 
 const defaultProps = {
-  onClick: () => {
-  },
+  onClick: undefined,
   type: 'primary',
   selector: 'radio',
+  icon: undefined,
+  children: undefined,
```

Comment:
[nit/optional] - TS should now be available in this repo, so if you wanted to, you could convert this file to TS and type the component out as a `React.FunctionComponent`. That should handle the `children` definition so you don't have to default it as `undefined`. Totally your call as converting to TS may cause additional ripple effects.

Reply:
I'm going to hold off on this, mostly because I don't feel totally comfortable in radiance yet and I don't want to break everything that uses it.